.github/workflows/samples-dotnet.yaml

```diff
@@ -1,4 +1,4 @@
-name: Samples C# .Net 7
```

Comment:
Consider renaming these workflows like samples-dotnet8.yaml and then moving aspnet servers to samples-dotnet6